### PR TITLE
tools: dump stores from etcd

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve? 
Sometimes, we need to check if the store meta info stored in etcd is same with PD.

### What is changed and how it works?
This PR adds a tool to dump the store meta info from etcd.

### Check List 

Tests 

 - Manual test 

Use tiup to start a cluster, then using this tool to dump the store info.

### Release note 
- Add a tool to dump store meta info from etcd

### CODE REVIEW

File: tools/stores-dump/main.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	endpoints = flag.String("endpoints", "http://127.0.0.1:2379", "endpoints urls")
+	filePath  = flag.String("file", "stores.dump", "dump file path and name")
+	caPath    = flag.String("cacert", "", "path of file that contains list of trusted SSL CAs")
+	certPath  = flag.String("cert", "", "path of file that contains X509 certificate in PEM format")
+	keyPath   = flag.String("key", "", "path of file that contains X509 key in PEM format")
+)
+
+const (
+	etcdTimeout = 1200 * time.Second
+
```
Comment (lhy1024) [inline tools/stores-dump/main.go:32]: unexpected empty line?

File: tools/stores-dump/main.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
+package main
```
Comment (disksing) [inline tools/stores-dump/main.go:1]: license

**Comment by rleungx** [comment]
@lhy1024 @disksing PTAL

**Comment by lhy1024** [comment]
LGTM

**Pull request merged**